### DISCUSSION TITLE
Edge-case megathread

### DISCUSSION THREAD

**Discussion by ItsVipra** [ItsVipra/ProToots#63] [General]
A place to leave all the edge-cases you find out in the wild.

**Comment by ItsVipra** [comment]
![image](https://github.com/ItsVipra/ProToots/assets/24212409/4837cb50-1ea3-4288-a9c7-73ee31bff1a1)

**Reply by nachtjasmin** [reply]
Is getting closed with #72.

**Comment by aurorasmiles** [comment]
![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/0092b360-581d-4175-af82-8082b9751797)
Might be easier to just search for any pronouns.page link in bio/tables, not sure how thats handled now

**Reply by nachtjasmin** [reply]
Is getting closed with #72.

**Comment by aurorasmiles** [comment]
![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/4c5c8f42-9cdc-47a1-8149-4172e4621f57)
in bio text

**Reply by nachtjasmin** [reply]
Going to be closed with #72 as well ^^

**Comment by aurorasmiles** [comment]
![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/42931fdf-0b72-497c-aa53-2d824776581f)
in table, might make sense to run the knownPronouns over those as well?

**Reply by nachtjasmin** [reply]
Going to be closed with #72.

**Comment by aurorasmiles** [comment]
![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/70030b6f-48d8-4467-aa98-4c21a281948b)
in bio, returns "e/em"

**Reply by nachtjasmin** [reply]
Going to be closed with #72.

**Comment by aurorasmiles** [comment]
![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/414c8865-b512-446e-8cc8-5964ff4fedc7)
in table

**Reply by nachtjasmin** [reply]
Going to be closed with #72.

**Comment by aurorasmiles** [comment]
![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/af8b64a5-ae39-445f-ab3b-73540631d3b9)
in table

**Reply by nachtjasmin** [reply]
Going to be closed with #72.

**Comment by ItsVipra** [comment]
![image](https://github.com/ItsVipra/ProToots/assets/24212409/679707e2-6c4a-4d50-9c2a-a889aa777501)
shown pronouns are `br`

**Reply by nachtjasmin** [reply]
Checked it against #72, it's working fine there.

**Comment by aurorasmiles** [comment]
Apparently you can put line breaks in your pronouns that don't get rendered in the profile, but do get rendered in ProToots
![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/1c71a789-39f3-4f1c-9cd3-f7772fb51940)

![grafik](https://github.com/ItsVipra/ProToots/assets/21148213/86d9be42-e7ea-4e15-ba0c-c3b182b02c54)

**Reply by nachtjasmin** [reply]
oh wow. that's a good catch. thanks for reporting it!
